platform/linux-generic/include/odp_schedule_scalable.h

```diff
@@ -138,6 +139,7 @@ typedef struct ODP_ALIGNED_CACHE {
 	bitset_t ODP_ALIGNED_CACHE rvec_free;
 	/* Reordering contexts to allocate from */
 	reorder_context_t ODP_ALIGNED_CACHE rvec[TS_RVEC_SIZE];
+	uint32_t loop_cnt; /*Loop counter to check pktio ingress queue dead loop */
```

Comment:
Line over 80 characters and space missing `*Loop`.

Reply:
ok , will reduce number .